nservicebus/logging/usage.md

```diff
@@ -10,6 +10,8 @@ redirects:
 Writing to logging from your code is straightforward. Set up a single static field to a `ILog` in your classes, and then use it in all your methods, like this:
 
 <!-- import UsingLogging -->
- 
 
-Note: Since `LogManager.GetLogger("Name");` is an expensive call it is important that the field is static so that the call only happens once per class.
+
+NOTE: Make sure that the log level is set to DEBUG when calling `.Debug(..)` or this log statement will not be added to the log file. See [Change settings via configuration](logging#changing-settings-via-configuration).
+
+NOTE: Since `LogManager.GetLogger(..);` is an expensive call it is important that the field is `readonly static` so that the call only happens once per class and have the best possible performance.
```

Comment:
using `readonly` is only valuable if you think a class is so complex that you will forget that a certain variable should not be modified. there is no performance improvement in using it. so dont bother adding it